### DISCUSSION TITLE
Is there a roadmap?

### DISCUSSION THREAD

**Discussion by matteobonin** [blazority/support#47] [Q&A] [answered]
I start by saying I really like this project and I love the fact that I basically use only c# and I have a  (almost) perfect result all the time. This is very good.

I was wondering if there was a roadmap for future implementations?

I know they are not _core components,_ but I think these could really help to speed up the development of enterprise apps:

**Grid**
https://clarity.design/angular-components/grid/

**Stepper**
https://clarity.design/angular-components/stepper/

**Layout**
https://clarity.design/storybook/core/?path=/story/layout-get-started--page

**Tables**
https://clarity.design/angular-components/table/
https://clarity.design/storybook/core/?path=/story/components-table--page  

**Select**
https://clarity.design/storybook/core/?path=/story/forms-select--page

Are there any thoughts/plans on these? Or even suggestions on how to approach a solution?

Thanks

PS

Keep up the good work!

**Comment by karan-kang** [comment, marked as answer]
Hi @matteobonin,

Thanks for your feedback, I am glad you liked the project.

Grid, and Stepper will be supported in the upcoming v0.10.0, releasing next week.
They are now tracked as feature issues here::
- [Stepper](https://github.com/blazority/support/issues/49)
- [Grid](https://github.com/blazority/support/issues/48)

We are still deciding whether the Table component should be ported.
The table implementation in Clarity is simple and clean, and works directly with natively supported HTML **table** tag.
You should be able to use it in your project as-is, let me know if you run into any challenges.
NOTE: A new version of table is currently in design with Clarity, preview is available in their [storybook](https://clarity.design/storybook/core/?path=/story/components-table--page).

As for Select, we already support this as shown here: https://blazority.com/select
Is there anything specific that was missed?

As for the Layout component, it is not yet published in their official docs. The storybook contains components that are still in design/draft stage. In the meantime, you should be able to achieve all of this by using display:flex or display:grid via CSS.

Hope this helps, thanks!

**Reply by matteobonin** [reply]
Hi @karan-kang 

for the select component I think it would be nice to have the multiple selection as if it was a list box?

see at the bottom:
https://clarity.design/storybook/core/?path=/story/forms-select--page

**Reply by matteobonin** [reply]
Hi @karan-kang 

for the table I think it kind of works, but surely the one in the preview is quite neat

**Comment by karan-kang** [comment]
Please use the combination of issue list/milestone as roadmap.
Closing
